### PR TITLE
Add

### PR DESCRIPTION
**Pull request by Ugboaja-Uchechi**
###  Project Description

- Create class Person with the following:

  Instance vars: @id, @name, and @age.
  Constructor with name, age, and parent_permission as parameter. name and parent_permission are optional and have default values of "Unknown" and true.
  Getters for @id, @name, and @age.
  Setters for @name and @age.
  Private method of_age? that returns true if @age is greater or equal to 18 and false otherwise.
  Public method can_use_services? that returns true if person is of age or if they have permission from parents.

- Create class Student with the following:

  Inherits from Person.
  Constructor extends parent's constructar by adding @classroom and a parameter for it.
  Method play_hooky that returns "¯\(ツ)/¯".

- Create class Teacher with the following:

  Inherits from Person.
  Constructor extends parent's constructar by adding @specialization and a parameter for it.
  Override can_use_services? so it always returns true.
  Each class should be saved in a separate file.

### CODE REVIEW

**Comment by Ugboaja-Uchechi** [comment]
I have worked on all project requirements to the best of my understanding.

No linter error.

File: README.md [hunk 1]
```diff
@@ -1 +1,45 @@
...
+## Built With
+
+This project was built with
+
+- Ruby
+
+Linter(s) such as:
+
+- Rubocop
+
```
Comment (mikethreels) [inline README.md]: - [x] please add information to your readme file on how to download your file to a local repo and how to run it as well.

File: person.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = id
```
Comment (mikethreels) [inline person.rb]: - [x] you are creating the instance variable `@id` and are adding the value of `id` however id is not passed as an argument, what you can do instead is assigning a random number to the instance variable `@id`

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Stephanie, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ugboaja-Uchechi** [comment]
Thank you for the reviews, I have implemented them.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Stephanie, 

Your project is complete! Great job for making the necessary changes recommended in the previous review. There is nothing else to say other than... it's time to merge it 🍾 :shipit:. Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**